projects/plugins/jetpack/modules/search/instant-search/lib/api.js

```diff
@@ -406,19 +388,19 @@ export function search( options, requestId ) {
 
 	const pathForPublicApi = `/sites/${ options.siteId }/search?${ queryString }`;
 
-	const { apiNonce, apiRoot, isPrivateSite, isWpcom } = window[ SERVER_OBJECT_NAME ];
-	if ( isPrivateSite && isWpcom ) {
-		return import( '../external/wpcom-proxy-request' ).then( ( { default: proxyRequest } ) => {
-			return promiseifedProxyRequest( proxyRequest, pathForPublicApi )
-				.catch( errorHandler )
-				.then( responseHandler );
-		} );
-	}
+	const { apiNonce, apiRoot, homeUrl, isPrivateSite, isWpcom } = window[ SERVER_OBJECT_NAME ];
 
-	// NOTE: Both atomic and Jetpack sites can be set to "private".
+	// NOTE: Both simple and atomic sites can be set to "private".
+	//       "Private" Jetpack sites are not yet supported.
 	const urlForPublicApi = `https://public-api.wordpress.com/rest/v1.3${ pathForPublicApi }`;
-	const urlForPrivateApi = `${ apiRoot }wpcom/v2/search?${ queryString }`;
-	const url = isPrivateSite ? urlForPrivateApi : urlForPublicApi;
+	const urlForWpcomOrigin = `${ homeUrl }/wp-json/wpcom-origin/v1.3${ pathForPublicApi }`;
```

Comment:
```suggestion
	const urlForWpcomOrigin = `${ apiRoot }wpcom-origin/v1.3${ pathForPublicApi }`;
```
would this work?

Reply:
No; for simple sites, the `apiRoot` is set to `public-api.wordpress.com`, which is not desirable here.